lib/element.ts

```diff
@@ -1061,7 +1061,7 @@ export class ElementFinder extends WebdriverWebElement {
  * @returns {ElementFinder} which identifies the located
  *     {@link webdriver.WebElement}
  */
-export let build$ = (element: ElementHelper, by: ProtractorBy) => {
+export let build$ = (element: ElementHelper, by: typeof By) => {
```

Comment:
Let's revert this to `ProtractorBy`. `ProtractorBy` is composed with `By` functionality and currently does not inherit from `By`.